### PR TITLE
bpo-40016: re docstring: Clarify relationship of inline and argument flags

### PR DESCRIPTION
**Pull request by cool-RR**
https://bugs.python.org/issue40016

### CODE REVIEW

File: Lib/re.py [hunk 1]
```diff
@@ -44,7 +44,8 @@
     "|"      A|B, creates an RE that will match either A or B.
     (...)    Matches the RE inside the parentheses.
              The contents can be retrieved or matched later in the string.
-    (?aiLmsux) Set the A, I, L, M, S, U, or X flag for the RE (see below).
+    (?aiLmsux) Apply flags to the entire pattern, allowing small tweaks
+               to the matching logic (details below).
```
Comment (terryjreedy) [inline Lib/re.py]: I gave a different replacement on the issue and added more above the flag list.

File: Misc/NEWS.d/next/Library/2020-03-19-19-40-27.bpo-40016.JWtxqJ.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Replace misleading uppercase flags in documentation
```
Comment (terryjreedy) [inline Misc/NEWS.d/next/Library/2020-03-19-19-40-27.bpo-40016.JWtxqJ.rst]: This is wrong.  The uppercase module constants are used for call arguments.  
```suggestion
In re docstring, clarify the relationship between inline and argument compile flags.
```

**Review by terryjreedy — CHANGES_REQUESTED** [review]
See issue.

**Comment by cool-RR** [comment]
I didn't expect the Spanish Inquisition

**Review by terryjreedy — APPROVED** [review]
Looks good.  The maxOS failure details were not available, so let us see what happens with retest.  Otherwise, I think this ready to merge.
---
Added: The new Docs/Docs failure makes no sense.  Git fetch failed and runner received a shutdown signal.  Failed again on re-run.  This patch does not touch the regular doc.  Even if this text includes blurbs, the change since the previous run should not cause a failure.

**Comment by terryjreedy** [comment]
@vstinner @pablogsal The Docs/Docs test is failing tonight with a git fetch failure.  The only 'doc change' is the news item.  Do either of you know what is going on?  May I merge anyway?

**Comment by terryjreedy** [comment]
The Travis checks passed, but this was not reported back here.

**Pull request merged**